source/Halibut.Tests/CancellationViaClientProxyFixture.cs

```diff
@@ -15,13 +16,16 @@ namespace Halibut.Tests
     public class CancellationViaClientProxyFixture : BaseTest
     {
         [Test]
-        public async Task CancellationCanBeDoneViaClientProxy()
+        [LatestClientAndLatestServiceTestCases(testNetworkConditions: false)]
+        [LatestClientAndPreviousServiceVersionsTestCases(testNetworkConditions: false)]
+        public async Task CancellationCanBeDoneViaClientProxy(ClientAndServiceTestCase clientAndServiceTestCase)
```

Comment:
clientAndServiceTestCase is unused

Reply:
lol

Reply:
Fixed that, also not that I added another test to this PR.